config/initializers/cors.rb

```diff
@@ -0,0 +1,15 @@
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+    allow do
+      origins '*'  # <- change this to allow requests from any domain while in development.
+  
+      resource '*',
+        headers: :any,
+        methods: [:get, :post, :put, :patch, :delete, :options, :head]
+    end
+  end
+  
```

Comment:
Are you sure you need cors?

Reply:
yes. it was erroring out when we ran it, so we added it in to fix the database errors. we added it with out mentors.

Reply:
We were having some CORs issues and followed some instructions they had found for how to skip the cors check. This bit of code was included in that description we followed.

Reply:
cors should be for separate frontend and backend applications. As a monolithic application cors shouldn't be doing anything for you. 

The skip_before_action :verify authenticity token should raise issues if you don't have it. Can you try to run the app with that command installed w/o cors and let me know what happens?

Reply:
hmmm okay

Reply:
I'd like sarah to get eyes on this. We'll be more available in a bit.